docker/cargo-runner.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Allows running tests inside docker. Takes care of linux-only dependencies and avoids
+# the need to emulate the architecture.
+
+set -o pipefail -eu
+
+PROGRAM=$1; shift
+
+main() {
+	local -r absolute_path="$(realpath ${PROGRAM})"
+
+	docker run \
```

Comment:
do we want to reuse test-utils crate for this?

Reply:
probably not imo